taskt/Core/Automation/User32/User32Functionas.cs

```diff
@@ -552,8 +552,10 @@ private static IntPtr MouseHookForLeftClickUpEvent(int nCode, IntPtr wParam, Int
                             UnhookWindowsHookEx(_mouseHookID);
                         }
 
-                        MSLLHOOKSTRUCT hookStruct = (MSLLHOOKSTRUCT)Marshal.PtrToStructure(lParam, typeof(MSLLHOOKSTRUCT));
```

Comment:
Was the hookstruct the cause of the offset issue? Is it worth leaving in?

Reply:
Yes, this was causing the offset. Originally I just forgot to delete the commented out lines but I'm not 100% sure the solution we found is reliable so I left it there in case we have to revert back to some version of the old method. I can delete it if you prefer.

Reply:
Lets leave it for now until we are sure it's stable.